docs/source/reservations/reservations.rst

```diff
@@ -170,32 +171,141 @@ In case you want to specify the DWS configuration for each job/cluster, you can
 
 .. _dws-on-gke:
 
-Using DWS on GKE with Kueue
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Using DWS on GKE
+~~~~~~~~~~~~~~~~~
+
+Flex-start, powered by `Dynamic Workload Scheduler <https://cloud.google.com/blog/products/compute/introducing-dynamic-workload-scheduler>`_, provides a flexible and cost-effective technique to obtain GPUs when you need to run AI/ML workloads.
+
+GKE supports the two kinds of flex-start configurations:
```

Comment:
Considering we don't expose the `mode` any more. Should we update the doc? We should not have the detailed explanation for flex-start here, considering we will automatically select. We can have the explanation in a dropdown

Reply:
Updated